src/v/config/configuration.cc

```diff
@@ -436,6 +436,27 @@ configuration::configuration()
       "truncation if any of the replicas has more data than the majority.",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       true)
+  , raft_max_inflight_follower_append_entries_requests_per_shard(
+      *this,
+      "raft_max_inflight_follower_append_entries_requests_per_shard",
+      "Maximum number of append entries requests, that were sent to a node "
+      "from a seastar shard, and that the current node is waiting for to be "
+      "replied. (previously this number was configured per raft group, now it "
+      "is configured per shard)",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      1024,
+      {.min = 1, .max = 50000})
+  , raft_max_buffered_follower_append_entries_bytes_per_shard(
+      *this,
+      "raft_max_buffered_follower_append_entries_bytes_per_shard",
+      "The total size of append entry requests that may be cached per shard, "
+      "using Raft buffered protocol. When an entry is cached the leader can "
+      "continue serving requests because the ordering of the cached requests "
```

Comment:
```suggestion
      "continue serving requests to followers because the ordering of the cached requests "
```

Reply:
not really, it can continue serving produce requests

Reply:
OK, will remove comment and leave as is.